### DISCUSSION TITLE
SSL Errors when setting up local VaultWarden with Rocket

### DISCUSSION THREAD

**Discussion by scepterus** [dani-garcia/vaultwarden#3270] [Q&A]
### Subject of the issue
I'm trying to self-host without a public certificate, by using the Rocket service and self-signed certificates.
I tried asking on the forum for help, but no one seems to know what is wrong. I set up everything according to the guide, however when I start the container I get:
`Rocket has launched from http://0.0.0.0:80`
but when trying to access the ip with https I get:
`ERR_SSL_PROTOCOL_ERROR` when I curl the link I get:
`curl: (35) OpenSSL/1.1.1s: error:1408F10B:SSL routines:ssl3_get_record:wrong version number`

### Deployment environment

/--------------------------------------------------------------------\
|                        Starting Vaultwarden                        |
|                           Version 1.27.0                           |
|--------------------------------------------------------------------|
| This is an *unofficial* Bitwarden implementation, DO NOT use the   |
| official channels to report bugs/features, regardless of client.   |
| Send usage/configuration questions or feature requests to:         |
|   https://vaultwarden.discourse.group/                             |
| Report suspected bugs/issues in the software itself at:            |
|   https://github.com/dani-garcia/vaultwarden/issues/new            |
\--------------------------------------------------------------------/
[INFO] No .env file found.
[2023-02-21 16:27:11.092][start][INFO] Rocket has launched from http://0.0.0.0:80
that's as far as I can get.

I started it from Docker with latest tag, via Portainer.
* Install method:

* Clients used: Chrome extension and android client

* Other relevant details:

### Steps to reproduce
just follow the guide on the wiki for self-signed, only use docker compose instead of run command. 

### Expected behaviour
I should reach the site on https and start setting up the server for use.

### Actual behaviour
When trying to access the server on HTTPS, I get blocked by SSL errors.

### Troubleshooting data
No logs yet, since haven't even set up the server beyond launch.

**Comment by stefan0xC** [comment]
I've answered in your original [forum post](https://vaultwarden.discourse.group/t/vaultwarden-with-rocket-gets-ssl-errors/2364/8?u=stefan0xc). Problem is the same that you have not configured the `ROCKET_TLS` environment variable.
